File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.java
class Solution {
    public int solution(int a, int b, int n) {
        int empty =n;
        int result =0;
        
        while(empty>=a){
            
            
            int rest = empty%a;
            int va = empty/a;
            result += va*b;

            empty = va*b + rest;
            
            

        }
        return result;
        
        
        
    }
}

File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.js
function solution(a, b, n) {
    let empty=n;
    let result =0;
    
    while(empty>=a)
        {
            let value= parseInt(empty/a);
            let rest = empty%a;
            
            empty = value*b+rest;
            result += value*b;
        
        }
    return result;
}